scipy/optimize/_differentialevolution.py

```diff
@@ -189,6 +209,17 @@ def differential_evolution(func, bounds, args=(), strategy='best1bin',
     >>> result.x, result.fun
     (array([1., 1., 1., 1., 1.]), 1.9216496320061384e-19)
 
+    Now repeat, but with parallelization (`aggressive` needs to be set to
+    False for this).
+
+    >>> import multiprocessing
+    >>> bounds = [(0,2), (0, 2), (0, 2), (0, 2), (0, 2)]
+    >>> with multiprocessing.Pool(2) as p:
```

Comment:
better to simplify the example now to just `workers=2`

Reply:
The example was meant to show that you could provide your own object (possessing a map method) for the evaluator. Perhaps it's better to have two examples. The first showing `workers=2`, the second showing how to provide your own `workers` object.